tests/unit/store-test.js

```diff
@@ -1157,3 +1157,57 @@ test("find with filter function returns array proxy that has a remove function t
   assert.equal(store._findById("person", 8).get("firstName"), "");
   assert.equal(store._findById("person", 8).get("lastName"), "new");
 });
+
+var Listing;
+
+moduleFor('service:simple-store', "store unit tests -- custom primary key", {
```

Comment:
One additional test worth writing would be an acceptance test (like the test below) to show the unsubscribe and filter api supports this. I realize it's a bit more work as you need to add a route/template to the dummy app :) but it's a nice add to ensure some regression tests for that unsubscribe / filter work.

https://github.com/toranb/ember-cli-simple-store/blob/master/tests/acceptance/filters-test.js#L39